docs/time.md

```diff
@@ -356,3 +356,17 @@ time = Temporal.Time.from('15:23:30.003');
 date = Temporal.Date.from('2006-08-24');
 time.withDate(date)  // => 2006-08-24T15:23:30.003
 ```
+
+### time.**getFields**() : { hour: number, minute: number, second: number, millisecond: number, microsecond: number, nanosecond: number }
+
+**Returns:** a plain object with properties equal to the fields of `time`.
+
+This method can be used to convert a `Temporal.Time` into a record-like data structure.
+It returns a new plain JavaScript object, with all the fields as enumerable, writable, own data properties.
+
```

Comment:
I think we're allowing time to be calendar-dependent, too.

Reply:
I think the last word on that was in #389, that days are always defined as solar days?

Reply:
Yeah, days are solar days, but do we allow calendars to customize how they tell time?  I've heard of calendars that have 4 day periods (morning, afternoon, evening, night) instead of 2 day periods (AM and PM).  The days are still solar days, but within the days, they don't use the same clock system for telling time.

Probably a topic for another issue.

Reply:
#522

Reply:
OK, thanks, I see what you meant now. I think it's indeed best to defer that to the issue you opened.